### PR TITLE
feat(sdk-coin-btc): send an inscription

### PR DESCRIPTION
**Pull request by davidkaplanbitgo**
Transfer an inscription from the current wallet to another address.

Similar to https://github.com/BitGo/BitGoJS/pull/3374 I will add tests after this merge, but need to unblock people

BG-68125

### CODE REVIEW

File: modules/sdk-coin-btc/src/inscriptionBuilder.ts [hunk 1]
```diff
@@ -29,6 +38,89 @@ export class InscriptionBuilder implements IInscriptionBuilder {
...
+    const rootWalletKeys = new utxolib.bitgo.RootWalletKeys(
+      wk.map((k) => {
+        assert(k.pub);
+        return utxolib.bip32.fromBase58(k.pub);
+      }) as Triple<utxolib.BIP32Interface>
+    );
+
+    const parsedSatPoint = parseSatPoint(satPoint);
+    const transaction = await this.wallet.getTransaction({ txHash: parsedSatPoint.txid });
+    const unspents = [transaction.outputs[parsedSatPoint.vout]];
```
Comment (OttoAllmendinger) [inline modules/sdk-coin-btc/src/inscriptionBuilder.ts:89]:
```suggestion
    // TODO(BG-70900): allow supplemental unspents
    const unspents = [transaction.outputs[parsedSatPoint.vout]];
```

**Review by OttoAllmendinger — DISMISSED** [review]
minor nits

**Review by OttoAllmendinger — APPROVED** [review]

**Review by JamesJo06 — DISMISSED** [review]

File: modules/sdk-coin-btc/src/inscriptionBuilder.ts [hunk 2]
```diff
@@ -29,6 +38,90 @@ export class InscriptionBuilder implements IInscriptionBuilder {
...
+      inscriptionConstraints: {
+        minChangeOutput?: bigint;
+        minInscriptionOutput?: bigint;
+        maxInscriptionOutput?: bigint;
+      };
+      changeAddressType: utxolib.bitgo.outputScripts.ScriptType2Of3;
+    } = {
+      signer: 'user',
+      cosigner: 'bitgo',
+      inscriptionConstraints: DefaultInscriptionConstraints,
```
[the remark this answers is not in the document]
Comment (OttoAllmendinger) [reply modules/sdk-coin-btc/src/inscriptionBuilder.ts:74]: good catch -  the defaults should be set in L58..L61

**Review by OttoAllmendinger — CHANGES_REQUESTED** [review]

File: modules/sdk-coin-btc/src/inscriptionBuilder.ts (hunk 2 above)
[the remark this answers is not in the document]
Comment (davidkaplanbitgo) [reply modules/sdk-coin-btc/src/inscriptionBuilder.ts]: should be good now

**Review by OttoAllmendinger — APPROVED** [review]

**Review by byhow — APPROVED** [review]

**Pull request merged**